### ISSUE TITLE
Update IP Allow List on Install

### ISSUE THREAD

**Issue by hismightiness** [cantarus/PolyDeploy#64]
It would be useful for the `IUpgradeable` feature to be added that automatically adds the current IP address that's installing the module so it doesn't need to be done via SQL in production environments.

**Comment by mitchelsellers** [comment]
Just a note on this one, I've tried this and it isn't necessarily consistent in the context of IUpgradable due to when it is triggered and how proxy or internal routing might work.  (At least that was my test.)

**Comment by hismightiness** [comment]
Ah.. Good point.  Maybe another method could be explored if we can't find something more consistent during IUpgradeable.

**Comment by mitchelsellers** [comment]
I would agree, totally great idea Will, just not sure of the best "accurate" way to make this happen.
